src/Accessor.tsx

```diff
@@ -0,0 +1,7 @@
+import { useCustomSelector } from "./redux/store";
+
+export default function Accessor(props: any) {
+  const { authToken } = useCustomSelector((state) => state.appointmentReducer);
```

Comment:
selectors should return the final data:
create selector like
const isUserLoggedIn = (state) => !!state.appointmentReducer.authToken;

const isAuth = useCustomSelector(isUserLoggedIn);